src/components/App.jsx

```diff
@@ -16,6 +17,12 @@ class App extends React.Component {
     this.props.dispatch(fetchProject());
   }
 
+  componentWillReceiveProps(nextProps) {
+    if (this.props.user != nextProps.user) {
```

Comment:
[minor?] This will trigger when the user has logged in, (`user` goes from `null` to an object) which is expected, but it'll also trigger when the user logs out, (`user` goes from an object to `null`) correct? Is this intended?

This is a minor comment at the moment because I've testing logging in/out and the app works fine. I do see a `401` error from seven-ten when I logout, but so far this hasn't caused any issues on our app itself. If seven-ten _does_ have an issue with asking for splits for a null user, however, this should be easily rectified by adding `...&& nextProps.user !== null` to that `if` clause.